src/components/AssetsTable/index.jsx

```diff
@@ -88,6 +95,33 @@ export class AssetsTable extends React.Component {
     });
   }
 
+  onLockClick(asset) {
+    this.props.toggleLockAsset(asset, this.props.courseDetails);
+  }
+
+  getImageThumbnail(thumbnail) {
+    const baseUrl = this.props.courseDetails.base_url ? this.props.courseDetails.base_url : '';
```

Comment:
Nit: you could consider `this.props.courseDetails.base_url || ''` instead

Reply:
yep, cleaner!